pnpm-lock.yaml

```diff
@@ -233,11 +233,15 @@ importers:
   packages/teams-js:
     specifiers:
       '@types/debug': ^4.1.7
+      '@types/dom-mediacapture-transform': 0.1.2
+      '@types/dom-webcodecs': 0.1.3
```

Comment:
I think this is an artifact from a previous build/install.

Reply:
removed